Vagrantfile

```diff
@@ -191,6 +191,15 @@ export SKIP_VENV_SHELL_WARNING=1
 # message after a successful run.
 set +x
 
+# Check if the zulip directory is writable
+if [ ! -w /srv/zulip ]; then
+    echo "Cannot write to Zulip directory. Run the following commands to fix this:"
+    # sudo is required since our uid is not 1000
+    echo "    rm -rf /PATH/TO/ZULIP/CLONE/.vagrant"
+    echo "    sudo chown -R 1000:$(whoami) /PATH/TO/ZULIP/CLONE"
+    echo "Replace /PATH/TO/ZULIP/CLONE with the path to where zulip code is cloned."
+    exit 1
```

Comment:
We should probably be clear that this needs to be run on the host, right?

And then also add something at the end telling the user to run `vagrant up` again, right?

Reply:
@timabbott Thanks for the quick review and the suggestions. 
I ended up trying this again, by following the instructions to the dot, and realized I was missing a `vagrant halt` step. If not, I get a port clash error since the failed attempt already setup some networking stuff (`redir`)

Also had to use single quotes (`'`) instead of double quotes - the `$(whoami)` gets evaluated otherwise.